### ISSUE TITLE
How about training DiMP from scratch?

### ISSUE THREAD

**Issue by shuida** [visionml/pytracking#178]
https://github.com/visionml/pytracking/blob/06d5d99c9a0265ee8e936db8698f5934c2a46ad2/ltr/train_settings/dimp/dimp18.py#L107
The target classification module and the estimation module are trained jointly in DiMP. You only fine-tune the pretrained backbone for DiMP. And the learning rates are different for each module. Why are the learning rate settings so complex? How about training them from scratch or using a same learning rate?

**Comment by martin-danelljan** [comment]
Hi. I dont think training a big ResNet from scratch here is very good. But we have only tried for some smaller feature backbones.

Yeah, the different learning rates are there due to some training instabilities that might appear if the DiMP module is give a too high learning rate. This has partly addressed, but we didnt change the settings so to keep them the same as we used in the paper.
